src/kili/services/copy_project/__init__.py

```diff
@@ -0,0 +1,139 @@
+        queries_project = QueriesProject(self.auth)
+        queries_project_user = QueriesProjectUser(self.auth)
+
+        self.projects = queries_project.projects
+        self.create_project = mutations_project.create_project
+        self.append_to_roles = mutations_project.append_to_roles
+        self.update_properties_in_project = mutations_project.update_properties_in_project
+        self.project_users = queries_project_user.project_users
+
+    def copy_project(  # pylint: disable=too-many-arguments
+        self,
+        from_project_id: str,
+        title: Optional[str] = None,
+        description: Optional[str] = None,
+        copy_json_interface: bool = True,
+        copy_quality_settings: bool = True,
+        copy_members: bool = True,
+    ) -> str:
+        """Copy an existing project.
+
+        Copy an existing source project from its ID.
+
+        Args:
+            from_project_id: Project ID to copy from.
+            title: Title for the new project. Defaults to source project
+                title if `None` is provided.
+            description: Description for the new project. Defaults to empty string
+                if `None` is provided.
+            copy_json_interface: Copy the json interface from the source project to the new one.
+            copy_quality_settings: Copy the quality settings from the source project to the new one.
+            copy_members: Copy the members from the source project to the new one.
+
+        Returns:
+            The created project ID.
+
+        Examples:
+            >>> kili.copy_project(from_project_id="clbqn56b331234567890l41c0")
+        """
+        fields = self.FIELDS_PROJECT
+        if copy_quality_settings:
+            fields = fields + self.FIELDS_QUALITY_SETTINGS
+
+        src_project = self.projects(  # type: ignore
+            project_id=from_project_id,
+            fields=fields,
+        )[0]
+
+        new_project_title = title
+        if new_project_title is None:
+            new_project_title = self._generate_project_title(src_title=src_project["title"])
+
+        new_project_description = description
+        if new_project_description is None:
+            new_project_description = ""
+
+        json_interface = {"jobs": {}}
+        if copy_json_interface:
+            json_interface = src_project["jsonInterface"]
+
+        new_project_id = self.create_project(
+            input_type=src_project["inputType"],
+            json_interface=json_interface,
+            title=new_project_title,
+            description=new_project_description,
+        )["id"]
+
+        if copy_members:
+            self._copy_members(from_project_id, new_project_id)
+
+        if copy_quality_settings:
+            self._copy_quality_settings(new_project_id, src_project)
+
+        return new_project_id
+
+    def _generate_project_title(self, src_title: str) -> str:
+        projects = self.projects(fields=["title"])
```

Comment:
Here you could query a lot of projects. We have a `searchQuery` parameter that you should use in the `ProjectWhere` type to find projects with a specific title. (The one that we use to filter projects by giving a project title on the project page on the app)